ui/ui-frontend/projects/referential/src/app/logbook-management-operation/logbook-management-operation-preview/logbook-management-operation-preview.component.html

```diff
@@ -107,3 +121,27 @@
     <button matDialogClose class="btn cancel link">{{ 'COMMON.CANCEL' | translate }}</button>
   </mat-dialog-actions>
 </ng-template>
+
+<ng-template #confirmCancelOperationDialog>
+  <mat-dialog-content class="dialog-content confirm-cancel">
```

Comment:
```suggestion
  <mat-dialog-content>
```
Ne pas définir de style particulier, quitte à ce que le Figma ne soit pas respecté, en attendant qu'on remette au propre les dialog : il ne sert à rien de définir des styles pour les mettre à la poubelle quelques jours/semaines plus tard (ou oublier de les supprimer).